modules/custom/openy_activity_finder/openy_cf_vue_app/yarn.lock

```diff
@@ -0,0 +1,5025 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
You are instructing to use npm in the README; but here we have yarn.lock - it doesn't look good. Please use npm, commit the package-lock.json and remove this yarn.lock file.

Reply:
updated README

Reply:
I would love to see some standardization here... npm vs yarn - npm is more common and we should have one way to simplify maintenance and CI environments. I would suggest to standardize on npm.
//cc @podarok